example-mlem-get-started/code/src/requirements.txt

```diff
@@ -0,0 +1,8 @@
+pandas
+scikit-learn
+scipy
+dvc[azure,s3]
+fastapi
+uvicorn
+docker
+mlem
```

Comment:
in the devcontainer we also install it, should we just install the requirements there?

Reply:
Removed the one from devcontainer.